Dnn.CommunityForums/class/Utilities.cs

```diff
@@ -825,14 +750,6 @@ public static string HTMLEncode(string strMessage = "")
 
             return strMessage;
         }
-
-        public static string ParsePre(string strMessage)
```

Comment:
This is removed... Is there a replacement?  Since it's public, shouldn't we follow a deprecation policy?